### PR TITLE
Fix multiple security group support

### LINKED ISSUE #10: Add support of multiple security groups
Part of #9

### PR DESCRIPTION
**Pull request by outcatcher**
Resolve #10 

Some minor changes

### CODE REVIEW

File: driver/opentelekomcloud_test.go [hunk 1]
```diff
@@ -155,3 +162,44 @@ func cleanupResources(driver *Driver) error {
...
+			"otc-cloud":       "otc",
+			"otc-subnet-name": subnetName,
+			"otc-vpc-name":    vpcName,
+			"otc-sec-groups":  sg.Name,
+		})
+	require.NoError(t, err)
+	require.NoError(t, driver.initCompute())
+	require.NoError(t, driver.initNetwork())
+	defer func() {
+		if err := cleanupResources(driver); err != nil {
```
Comment (lego963) [inline driver/opentelekomcloud_test.go]: `defer func() {`
		`assert.NoError(t, cleanupResources(driver))`
	`}()`
Will we have 2 defer statement styles?

File: driver/services/compute_test.go [hunk 2]
```diff
@@ -100,9 +99,9 @@ func TestClient_CreateSecurityGroup(t *testing.T) {
 	sg, err := client.CreateSecurityGroup(sgName, 22)
```
Comment (lego963) [inline driver/services/compute_test.go]: Don't we need here any defer?

Comment (outcatcher) [reply driver/services/compute_test.go]: Not really, there is no `require.` calls after that, so `assert.NoError(t, client.DeleteSecurityGroup(sg.ID))` will be executed any way

File: driver/opentelekomcloud_test.go (hunk 1 above)
Comment (outcatcher) [reply driver/opentelekomcloud_test.go]: Fixed

**Pull request merged**